app/components/related_links/edit_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module RelatedLinks
```

Comment:
I was thinking we'd treat this as a separate but related component in case we wound up using related links for collections, but maybe that's not a need?

Reply:
You're right, collections have related links as well.

Reply:
OK, will leave this component outside the works structure then